spec/support/cohortManager.spec.js

```diff
@@ -0,0 +1,103 @@
+const CohortManager = require('../../src/cohortManager')
+
+describe('Test cohort manager', () => {
+  let cohortManager
+
+  beforeEach(() => {
+    cohortManager = new CohortManager()
+  })
+
+  it('should create a cohort name', () => {
+    // set up
+    const namesCohort = 'Cohort 10'
+    // execute
+    const cohort = cohortManager.createCohort(namesCohort)
+
+    // verify
+    expect(cohortManager.cohorts.length).toBe(1)
+    expect(cohort.name).toBe(namesCohort)
+    expect(cohort.students).toEqual([])
+  })
+
+  fit('adds a student to an existing cohort', () => {
```

Comment:
Your src/cohortManager.js line 77 is returning student - do you expect this to be returning an object? You could maybe try writing the object in your test and expecting it to be equal to that rather than having expectations for each parameter?